monitors/ContractMonitor.js

```diff
@@ -27,6 +27,9 @@ class ContractMonitor {
     this.collateralCurrencySymbol = "DAI";
     this.syntheticCurrencySymbol = "ETHBTC";
 
+    // TODO: pull this into the parent client
+    this.networkId = 1;
```

Comment:
What's the parent client? How intrusive will this be to implement? If it's not that bad, can we just go ahead and do it in this PR or in a quick follow-up so we don't forget?

Reply:
I think I should add it in the same PR as this issue: #1417. If you prefer I can do it here but it's the same kind of refactor that that issue needs.

Reply:
SGTM